diagram.md

```diff
@@ -89,9 +89,11 @@ The `Info Element` contains vital information for identifying the whole `Segment
 |      | WritingApp       |
 |-------------------------|
 ```
-Figure: Representation of a `Info Element` and its `Child Elements`.
+Figure: Representation of an `Info Element` and its `Child Elements`.
 
-The `Tracks Elements` tells us the technical details of what is in each track. For instance, is it a video, audio or subtitle track? What resolution is the video? What sample rate is the audio? The `Tracks Elements` can store the name, number, unique identifier, language, and type (audio, video, subtitles, etc) of each track. The `Tracks Element` also identifies what codec to use to decode the track and has the codec's private data for the track.
+The `Tracks Element` defines the technical details for each track and can store the name, number, unique identifier, language and type (audio, video, subtitles, etc.) of each track. For example, the `Tracks Element` MAY store information about the resolution of a video track or sample rate of an audio track.
+
+The `Tracks Element` MUST identify all the data needed by the codec to decode the data of the specified track. However, the data required is contingent on the codec used for the track. For example, a `Track Element` for uncompressed audio only requires the audio bit rate to be present. A codec such as AC-3 would require that the `CodecID Element` be present for all tracks, as it is the primary way to identify which codec to use to decode the track.
```

Comment:
My question was about usability (i.e. find all information at one place)… but nothing beyond that!